packages/lesswrong/components/sunshineDashboard/SunshineNewUsersItem.jsx

```diff
@@ -0,0 +1,120 @@
+import { Components, registerComponent, withEdit, withCurrentUser } from 'meteor/vulcan:core';
+import React, { Component } from 'react';
+import Users from 'meteor/vulcan:users';
+import { Link } from 'react-router'
+import FontIcon from 'material-ui/FontIcon';
+import moment from 'moment';
+import PropTypes from 'prop-types';
+
+class SunshineNewUsersItem extends Component {
+
+  handleReview = () => {
+    const { currentUser, user, editMutation } = this.props
+    editMutation({
+      documentId: user._id,
+      set: {reviewedByUserId: currentUser._id},
+      unset: {}
+    })
+  }
+
+  handlePurge = () => {
+    const { currentUser, user, editMutation } = this.props
+    if (confirm("Are you sure you want to delete all this user's posts, comments and votes?")) {
```

Comment:
Add `\*global confirm*\`at the top, to not confuse people where `confirm` comes from